compatible/backup_cluster.yaml

```diff
@@ -0,0 +1,204 @@
+    # ulimits:
+    #   nofile:
+    #     soft: 1000000
+    #     hard: 1000000
+    restart: on-failure
+    env_file:
+      - ../docker/minio.env
+
+  tikv1:
+    image: pingcap/tikv:${TAG}
+    volumes:
+      - ../docker/config/tikv.toml:/tikv.toml:ro
+      - /tmp/br/docker/backup_data/${TAG}:/data
+      - /tmp/br/docker/backup_logs/${TAG}:/logs
+    command:
+      - --addr=0.0.0.0:20160
+      - --advertise-addr=tikv1:20160
+      - --data-dir=/data/tikv1
+      - --pd=pd0:2379
+      - --config=/tikv.toml
+      - --log-file=/logs/tikv1.log
+    depends_on:
+      - "pd0"
+    # sysctls:
+    #   net.core.somaxconn: 32768
+    # ulimits:
+    #   nofile:
+    #     soft: 1000000
+    #     hard: 1000000
+    restart: on-failure
+    env_file:
+      - ../docker/minio.env
+
+  tikv2:
+    image: pingcap/tikv:${TAG}
+    volumes:
+      - ../docker/config/tikv.toml:/tikv.toml:ro
+      - /tmp/br/docker/backup_data/${TAG}:/data
+      - /tmp/br/docker/backup_logs/${TAG}:/logs
+    command:
+      - --addr=0.0.0.0:20160
+      - --advertise-addr=tikv2:20160
+      - --data-dir=/data/tikv2
+      - --pd=pd0:2379
+      - --config=/tikv.toml
+      - --log-file=/logs/tikv2.log
+    depends_on:
+      - "pd0"
+    # sysctls:
+    #   net.core.somaxconn: 32768
+    # ulimits:
+    #   nofile:
+    #     soft: 1000000
+    #     hard: 1000000
+    restart: on-failure
+    env_file:
+      - ../docker/minio.env
+
+  tikv3:
+    image: pingcap/tikv:${TAG}
+    volumes:
+      - ../docker/config/tikv.toml:/tikv.toml:ro
+      - /tmp/br/docker/backup_data/${TAG}:/data
+      - /tmp/br/docker/backup_logs/${TAG}:/logs
+    command:
+      - --addr=0.0.0.0:20160
+      - --advertise-addr=tikv3:20160
+      - --data-dir=/data/tikv3
+      - --pd=pd0:2379
+      - --config=/tikv.toml
+      - --log-file=/logs/tikv3.log
+    depends_on:
+      - "pd0"
+    # sysctls:
+    #   net.core.somaxconn: 32768
+    # ulimits:
+    #   nofile:
+    #     soft: 1000000
+    #     hard: 1000000
+    restart: on-failure
+    env_file:
+      - ../docker/minio.env
+
+  tidb:
+    image: pingcap/tidb:${TAG}
+    ports:
+      - 400${PORT_SUFFIX}:4000
+      - 1008${PORT_SUFFIX}:10080
+    volumes:
+      - ../docker/config/tidb.toml:/tidb.toml:ro
+      - /tmp/br/docker/backup_logs/${TAG}:/logs
+      - ./credentials:/credentials
+    command:
+      - --store=tikv
+      - --path=pd0:2379
+      - --config=/tidb.toml
+      - --log-file=/logs/tidb.log
+      - --advertise-address=tidb
+    depends_on:
+      - "tikv0"
+      - "tikv1"
+      - "tikv2"
+      - "tikv3"
+    # sysctls:
+    #   net.core.somaxconn: 32768
+    # ulimits:
+    #   nofile:
+    #     soft: 1000000
+    #     hard: 1000000
+    restart: on-failure
+    env_file:
+      - ../docker/gcs.env
+
+  minio:
+    image: minio/minio
+    ports:
+      - 2492${PORT_SUFFIX}:24927
+    volumes:
+      - /tmp/br/docker/data/s3:/data/s3
+    command: server --address=:24927 /data/s3
```

Comment:
If we have used arrays for `command` all the times previously, I think maybe keep consistence would be better?

Reply:
Done